docs/usage/features.md

```diff
@@ -48,6 +48,7 @@ option of nfd-worker.
 | ----------------------- | ------------ | -----------
 | **`cpu-cpuid.<cpuid-flag>`**      | true   | CPU capability is supported. **NOTE:** the capability might be supported but not enabled.
 | **`cpu-hardware_multithreading`** | true   | Hardware multithreading, such as Intel HTT, enabled (number of logical CPUs is greater than physical CPUs)
+| **`cpu-nx_gzip`**                 | true   | Nest Accelerator for GZIP is supported(Power).
```

Comment:
I think we could use the same `coprocessor` "sub-cpu-feature-prefix" as for the raw feature.

```suggestion
| **`cpu-coprocessor.nx_gzip`**                 | true   | Nest Accelerator for GZIP is supported(Power).
```